<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outofbounds</class>
 <widget class="QWidget" name="outofbounds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QLabel" name="field">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>230</y>
     <width>601</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/new/prefix1/Images/soccerField3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sideLineOutOfBounds2">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>590</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Sideline:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If the ball completely crosses the sideline whether in ground or air and enters this area, this ball goes out of bounds.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Result: &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;a throw-in is granted to the team that did not touch the ball last&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Rules for throw-in: &lt;/span&gt;Both feet must be on or behind thesideline, touching the ground. The ball must be thrown from behind and over the head&lt;span style=&quot; font-weight:700;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/Images/sideline.png</normaloff>:/new/prefix1/Images/sideline.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>605</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="behindGoalOutOfBounds1">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>260</y>
     <width>51</width>
     <height>331</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Goal Line (Not a goal):&lt;/span&gt;&lt;/p&gt;&lt;p&gt;When the ball &lt;span style=&quot; font-weight:700;&quot;&gt;completely crosses the goal line&lt;/span&gt; and eneter this area but &lt;span style=&quot; font-weight:700;&quot;&gt;no goal is scored&lt;/span&gt;, one of two things can happen:&lt;/p&gt;&lt;h5 style=&quot; margin-top:12px; margin-bottom:4px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:small; font-weight:700;&quot;&gt;a. Last touched by the attacking team&lt;/span&gt;&lt;/h5&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restart: Goal kick&lt;/span&gt; for the defending team.&lt;/li&gt;&lt;/ul&gt;&lt;h5 style=&quot; margin-top:12px; margin-bottom:4px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:small; font-weight:700;&quot;&gt;b. Last touched by the defending team&lt;/span&gt;&lt;/h5&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restart: Corner kick&lt;/span&gt; for the attacking team. &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/Images/goalLine1.png</normaloff>:/new/prefix1/Images/goalLine1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>331</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius: 30px;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:700; text-decoration: underline;&quot;&gt;Ball out of bounds&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;The ball goes out of bounds if it goes past the boundaries of the field, marked by the white lines at the edges. Hover over the areas outside the edges to know more about what happens when the ball goes out of bounds. &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Click on the areas outside the edges to see what happens when the ball goes in that area.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sideLineOutOfBounds1">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>230</y>
     <width>599</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Sideline:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If the ball completely crosses the sideline whether in ground or air and enters this area, this ball goes out of bounds.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Result: &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;a throw-in is granted to the team that did not touch the ball last&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Rules for throw-in: &lt;/span&gt;Both feet must be on or behind thesideline, touching the ground. The ball must be thrown from behind and over the head&lt;span style=&quot; font-weight:700;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/Images/sideline.png</normaloff>:/new/prefix1/Images/sideline.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>605</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="behindGoalOutOfBounds2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>51</width>
     <height>331</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Goal Line (Not a goal):&lt;/span&gt;&lt;/p&gt;&lt;p&gt;When the ball &lt;span style=&quot; font-weight:700;&quot;&gt;completely crosses the goal line&lt;/span&gt; and eneter this area but &lt;span style=&quot; font-weight:700;&quot;&gt;no goal is scored&lt;/span&gt;, one of two things can happen:&lt;/p&gt;&lt;h5 style=&quot; margin-top:12px; margin-bottom:4px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:small; font-weight:700;&quot;&gt;a. Last touched by the attacking team&lt;/span&gt;&lt;/h5&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restart: Goal kick&lt;/span&gt; for the defending team.&lt;/li&gt;&lt;/ul&gt;&lt;h5 style=&quot; margin-top:12px; margin-bottom:4px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:small; font-weight:700;&quot;&gt;b. Last touched by the defending team&lt;/span&gt;&lt;/h5&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restart: Corner kick&lt;/span&gt; for the attacking team. &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/Images/goalLine2.png</normaloff>:/new/prefix1/Images/goalLine2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>331</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
